### ISSUE TITLE
BoF implementation code 

### ISSUE THREAD

**Issue by abhigoku10** [dmlc/gluon-cv#1147]
@mli @piiswrong @larroy @leezu @feynmanliang  where can i find the implementation of "Bag of Freebies for Training Object Detection Neural Networks" code implementation Thanks in adavace

**Comment by larroy** [comment]
@zhreshold

**Comment by zhreshold** [comment]
@abhigoku10 Please checkout https://github.com/dmlc/gluon-cv/blob/master/scripts/detection/yolo/train_yolo3.py with corresponding training hyperparameters https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/detection/yolo3_darknet53_coco.sh.

You can always find more in this page: https://gluon-cv.mxnet.io/model_zoo/detection.html#id81

**Comment by sudo-rm-covid19** [comment]
> @abhigoku10 Please checkout https://github.com/dmlc/gluon-cv/blob/master/scripts/detection/yolo/train_yolo3.py with corresponding training hyperparameters https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/detection/yolo3_darknet53_coco.sh.
> 
> You can always find more in this page: https://gluon-cv.mxnet.io/model_zoo/detection.html#id81

Hi @zhreshold ,
Cosine learning rate seems to be missing in the script. Are there updated scripts to reproduce the results in BoF paper?
Thanks.
